src/js/App/Header/AppFilter.js

```diff
@@ -0,0 +1,114 @@
+import React, { useState, useEffect } from 'react';
+import { safeLoad } from 'js-yaml';
+import { Dropdown } from '@patternfly/react-core/dist/js/components/Dropdown/Dropdown';
+import { DropdownToggle } from '@patternfly/react-core/dist/js/components/Dropdown/DropdownToggle';
+import { CaretDownIcon } from '@patternfly/react-icons/dist/js/icons/caret-down-icon';
+import { SearchInput } from '@patternfly/react-core/dist/js/components/SearchInput/SearchInput';
+import { TextContent } from '@patternfly/react-core/dist/js/components/Text/TextContent';
+import { Text, TextVariants } from '@patternfly/react-core/dist/js/components/Text/Text';
+import { Grid } from '@patternfly/react-core/dist/js/layouts/Grid/Grid';
+import { GridItem } from '@patternfly/react-core/dist/js/layouts/Grid/GridItem';
+
+import CogIcon from '@patternfly/react-icons/dist/js/icons/cog-icon';
+import ansible from '../../../../static/images/platform-icons/ansible.svg';
+import openshift from '../../../../static/images/platform-icons/openshift.svg';
+import costManagement from '../../../../static/images/platform-icons/cost-management.svg';
+import migrationsNamespace from '../../../../static/images/platform-icons/migrations-namespace.svg';
+
+import { getNavFromConfig } from '../../nav/globalNav';
+import sourceOfTruth from '../../nav/sourceOfTruth';
+
+import './AppFilter.scss';
+
+const getIcon = (id = 'default') =>
+  ({
+    insights: <img className="ins-c-app-filter-app-icon" src="/apps/landing/fonts/Insights.svg" alt="Insights Logo" />,
```

Comment:
I would add `isBeta` check in here.

```suggestion
    insights: <img className="ins-c-app-filter-app-icon" src={`${
      insights.chrome.isBeta() ?
      'beta' :
      ''
    }/apps/landing/fonts/Insights.svg" alt="Insights Logo`} />,
```